ui/js/controllers/perf/compare.js

```diff
@@ -263,9 +272,9 @@ perf.controller('CompareResultsCtrl', [
             $scope.testList = Object.keys($scope.compareResults).sort().concat([noiseMetricTestName]);
             $scope.titles[noiseMetricTestName] = noiseMetricTestName;
 
-            // call digest explicitly so we don't have to worry about when promises
+            // call $apply explicitly so we don't have to worry about when promises
             // get resolved (see bug 1470600)
-            $scope.$digest();
+            $scope.$apply();
```

Comment:
I had to look up the difference between `$apply` and `$digest`, and in so doing came across this, which filled in a few gaps I had:
http://jimhoskins.com/2012/12/17/angularjs-and-apply.html

I guess the lack of error handling for `fn(); $apply();` vs `$apply(fn());` doesn't really affect us? (/we're not going to need these for too much longer anyway..)

Reply:
I had to look that up myself before.  :)  This was a nice write-up.  And, yeah, may have been better to wrap the model calls with ``$apply(fn())`` instead.  But, as you say, this code is all on the chopping block anyway.  Sarah will be doing the conversion, as I understand it.  So, yeah... Thankfully, this will soon all be moot.  :)